app/extensions/helper/helpers.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
This is in the wrong place. lithium extensions are for extending a base lithium class.
Put this in \nzedb\utilities\misc.php for now